frontend/sqlscripting/mpala_migration.sql

```diff
@@ -0,0 +1,127 @@
+SET foreign_key_checks = 1;
```

Comment:
For reusable SQL scripts, add a comment at the top or place an adjacent README explaining what they are. Otherwise remove if the scripts have no future value.

Reply:
I've pulled them! took a little bit to figure out how to delete the directories from the remote repo too, but they've been added to the gitignore so shouldn't get added back in future iterations either!